libr/bin/p/bin_dex.c

```diff
@@ -2104,6 +2115,11 @@ static int cmp_path(const void *a, const void *b) {
 	return strcmp ((const char*)a, (const char*)b);
 }
 
+static bool is_classes_dex(const char *filename) {
+	return r_str_startswith (filename, "classes") \
```

Comment:
IMHO this splitting just makes reading more complicate. Without it the line wouldn't really be that long.

Reply:
there's no need to mess a conditional with two function calls and handcrafted strings, when the function name can clearly specify what is going to check in there. it is more readable imho